tcms/static/js/testrun_actions.js

```diff
@@ -551,6 +551,26 @@ function changeCaseRunAssignee() {
   });
 }
 
+function updateCaseRunText() {
+  var executions = serializeCaseRunFromInputList(jQ('#id_table_cases')[0]);
+  if (!executions.length) {
+    window.alert(default_messages.alert.no_case_selected);
+    return false;
+  }
+
+  executions.forEach(execution_id =>
+    jsonRPC('TestExecution.update', [execution_id, {'case_text_version': getTextVersionByExecutionId(execution_id)}], () => { }, sync=true)
+  );
+  window.location.reload(true);
+}
+
+function getTextVersionByExecutionId(execution_id) {
+    return jQ('#id_table_cases tbody tr')
+    .has('td .caserun_selector')
+    .has('input[value=' + execution_id + ']')
+    .find('input[name="case_text_version"]').val();
+}
```

Comment:
this is useless, because it tells you the current value of the `case_text_version`. depending on which approach you chose, you will either not need this at all, or would need to rework it